deployment-platforms/google-cloud-functions/README.md

```diff
@@ -0,0 +1,19 @@
+# Google Cloud Functions
```

Comment:
I don't know why this is here

Reply:
This happens because https://github.com/prisma/prisma-examples/tree/master/.github/readmes/deployment-platforms/google-cloud-functions is leftover from when some deployments examples were removed: https://github.com/prisma/prisma-examples/commit/7408f5640a82967da8e0efc81123fc6f43f7695e#diff-e4636cb1a412c419eebfc668bf77329f Can https://github.com/prisma/prisma-examples/tree/master/.github/readmes/deployment-platforms/google-cloud-functions be safely removed @nikolasburk?

Reply:
Ah yes, sorry for about the confusion here! The deployment examples are not a thing any more (for now), so these leftover READMEs can be deleted.